Source/Core/DolphinQt2/MainWindow.cpp

```diff
@@ -195,6 +195,10 @@ void MainWindow::ConnectMenuBar()
   connect(m_menu_bar, &MenuBar::ShowList, m_game_list, &GameList::SetListView);
   connect(m_menu_bar, &MenuBar::ColumnVisibilityToggled, m_game_list,
           &GameList::OnColumnVisibilityToggled);
+  connect(m_menu_bar, &MenuBar::GameListPlatformVisibilityToggled, m_game_list,
+    &GameList::OnGameListPlatformVisibilityToggled);
+  connect(m_menu_bar, &MenuBar::GameListRegionVisibilityToggled, m_game_list,
```

Comment:
I suggest connecting both of these signals to the same function, as they're identical, as far as I can tell.